src/components/calculator.js

```diff
@@ -1,32 +1,36 @@
 import React from 'react';
+import {
+  numberClick, operatorClick, ac, dot, result, porcentaje,
+} from '../modules/calculate';
 
 const Calculator = () => (
   <div className="calculator">
     <h2>Lets do some math </h2>
     <div className="calculator-box">
       <form>
         <div className="grid">
-          <input type="text" value="0" readOnly />
+          <input className="input" type="text" value="0" readOnly />
+          <p className="saved">0</p>
           <div className="buttons">
```

Comment:
- [ ] As per requirements:
- Note: Practice what you have learned until now: use useState and props.
, kindly consider using `useState` in this Calculator component for representing changes in the output by mathematic operations. you can implement this logic in your component like below:
```
  const [state, setState] = useState({
    total: 0,
    next: null,
    operation: null,
  });

  const handleClick = (event) => {
    const result = calculate(state, event.target.innerText);
    return setState(result);
  };
```